yi-language/package.yaml

```diff
@@ -0,0 +1,67 @@
+name: yi-language
+version: '0.2.1'
+synopsis: Collection of language-related Yi libraries.
+maintainer: Dmitry Ivanov <ethercrow@gmail.com>
+license: GPL-2
+github: yi-editor/yi
+category: Yi
+
+ghc-options: -Wall -ferror-spans
+
+#dependencies:
```

Comment:
If I uncomment this part and remove the dependencies in the `library` part, I generate a correct cabal file.